### ISSUE TITLE
7.1.10. Screen Name/Title: Creating an account - Missing role="dialog" or role="alertdialog"

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-admin#2325]
### Component: Read and agree to the terms of use - modal window
Observation:
Missing role="dialog" or role="alertdialog"
The modal is not announced by screen readers as a dialog/modal because it lacks the proper ARIA role.

(Attachment: 7.1.9. Modal dialog not announced or structured accessibly.mp4)

**Recommendation:** 
Example:
Add role="dialog" to the outer modal container:
<div id="tou-terms" role="dialog" ...>
